lib/gollum/app.rb

```diff
@@ -348,7 +348,7 @@ def wiki_new
         wiki.write_page(name, format, params[:content], commit_message, path)
 
         page_dir = settings.wiki_options[:page_file_dir].to_s
-        redirect to("/#{clean_url(::File.join(page_dir, path, encodeURIComponent(name)))}")
+        redirect to("/#{clean_url(::File.join(encodeURIComponent(page_dir), encodeURIComponent(path), encodeURIComponent(name)))}")
```

Comment:
Sorry guys, I didn't answered because I wasn't available at all :(
Glad you finally solved and merged it :)

Thanks for your time